CONTRIBUTING.rst

```diff
@@ -134,34 +134,23 @@ Documentation
 C++
 ---
 
-.clang-format
-~~~~~~~~~~~~~
-
-* When starting a new C++ project, copy the .clang-format file included here.
-* Our style is based on Google (as opposed to LLVM, Chromium, Mozilla, or Webkit) with minor differences.
-* :code:`ColumnLimit` is set to :code:`120`, as specified in `Coding Conventions`_.
-* :code:`NamespaceIndentation` is set to :code:`Inner` as a middle ground between :code:`None` (Google) and :code:`All`,
-  such that every line in a file defining a namespace isn't indented, but nested namespaces are easily spotted.
-* Various indent-width specifiers are scaled by a factor of 2 such that the base indent is :code:`4`, as specified in
-  `Coding Conventions`_, instead of :code:`2` (Google). This is especially helpful for readibility in cases like
+C++ Version
+~~~~~~~~~~~
 
-  .. code-block:: c++
+C++ code should be compatible with standard C++11.
 
-    if (condition) {
-        foo();
-    } else {
-        bar();
-    }
+Naming
+~~~~~~
+* File names should be lowercase with underscores and end with ``.cpp`` or ``.hpp``.
+* Type names should be PascalCase. i.e. :code:`AdjArrayBQM`
+* Function names should be lowercase with underscores. i.e. :code:`num_variables()`.
+* Variables names should be lowercase with underscores. Private data members should have a trailing underscore.
+* Macros should be ``ALL_CAPS_WITH_UNDERSCORES``.
 
-  as opposed to
+Format
+~~~~~~
 
-  .. code-block:: c++
-
-    if (condition) {
-      foo();
-    } else {
-      bar();
-    }
+When starting a new C++ project, use clang-format with the .clang-format file included here.
```

Comment:
Worth pointing out we base our style on [Google's](https://google.github.io/styleguide/cppguide.html). Also linking to it.